diff --git a/components/password_manager/core/browser/login_database_mac.cc b/components/password_manager/core/browser/login_database_mac.cc
index 43d3736886ceacf8cb3787382304db404cfc084c..5e1ff136dce58805d441fb7b7ae15d422297b2fe 100644
--- a/components/password_manager/core/browser/login_database_mac.cc
+++ b/components/password_manager/core/browser/login_database_mac.cc
@@ -25,7 +25,7 @@ LoginDatabase::EncryptionResult LoginDatabase::DecryptedString(
   if (OSCrypt::DecryptString16(cipher_text, plain_text))
     return ENCRYPTION_RESULT_SUCCESS;
 
-  return OSCrypt::IsEncryptionAvailable() && IsUsingCleanupMechanism()
+  return OSCrypt::IsEncryptionAvailable() /* && IsUsingCleanupMechanism() */
              ? ENCRYPTION_RESULT_ITEM_FAILURE
              : ENCRYPTION_RESULT_SERVICE_FAILURE;
 }
